### PR TITLE
[CRI-1211] Update project structure to more closely resemble other public repos

### PR DESCRIPTION
**Pull request by bryantjustin**
#### Ticket

[CRI-1211](https://vimean.atlassian.net/browse/CRI-1211)

#### Pull Request Checklist

- [x] Resolved any merge conflicts
- [x] No build errors or warnings are introduced
- [x] New files are written in Swift
- [x] New classes contain license headers
- [x] New classes have Documentation
- [x] New public methods have Documentation

#### Issue Summary

- Reorganized the repo to be structured similar to other public repos

#### Implementation Summary

- Moved code into `Sources`.
- Moved examples into `Examples`.
- Updated `Podspec` to point to correct source directory.
- Updated `Podfile` to reflect new project structure and to be more inline with other repos.

#### Reviewer Tips

- N/A

#### How to Test

- New project compiles
- iOS project on `develop` compiles against this branch

### CODE REVIEW

**Comment by cameo-github** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Capabilities for Example may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the build settings editor. (in target ‘Example’)</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.002) seconds</td>
    </tr>
  </tbody>
</table>

See build details on [CircleCI](https://circleci.com/gh/vimeo/PlayerKit/140)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: fastlane/Fastfile [hunk 1]
```diff
@@ -24,12 +24,12 @@ platform :ios do
   lane :test do |options|
 
     cocoapods(
```
Comment (rpassis) [inline fastlane/Fastfile]: the `podfile` arg is redundant here - we can just use `cocoapods`

**Review by rpassis — COMMENTED** [review]
So I only have a minor comment on the Fastfile, ~and also a question around whether we should check the Pods in or not. I personally think we should and this would also allow us to support Carthage out of the box~.

~And on that note if we decide to check in the Pods folder one small thing you will need to do to support Carthage is go to manage schemes, and uncheck then check again the PlayerKit-iOS shared checkbox (yes, Xcode).~ 

~This should result in a dirty repo with a new xcscheme file added. Once this file is checked in we are good to go.~

Scratch that, I just pulled your last commit and that's done.

**Review by rpassis — APPROVED** [review]
LGTM, thanks for your first contribution 😜

**Comment by bryantjustin** [comment]
@rpassis I removed the `cocoapods` command instead of just the `podfile` arg. I amended the previous commit. Didn't think you'd review it so quickly!

**Comment by bryantjustin** [comment]
@rpassis as per our conversation, I removed it again. 🙃

**Review by rpassis — APPROVED** [review]

**Comment by rpassis** [comment]
Looks good once the build is green.

**Pull request merged**